src/state/slices/opportunitiesSlice/selectors/combined.ts

```diff
@@ -142,6 +145,39 @@ export const selectAggregatedEarnOpportunitiesByAssetId = createDeepEqualOutputS
   },
 )
 
+export const selectClaimableRewards = createDeepEqualOutputSelector(
+  selectUserStakingOpportunitiesWithMetadataByFilter,
+  selectAssets,
+  selectMarketDataSortedByMarketCap,
+  (userStakingOpportunitesWithMetadata, assets, marketData): string => {
+    return userStakingOpportunitesWithMetadata
+      .reduce<BN>((sum, stakingOpportunityWithMetadata) => {
+        const rewardsAmountFiat = Array.from(
+          stakingOpportunityWithMetadata?.rewardAssetIds ?? [],
+        ).reduce((sum, assetId, index) => {
```

Comment:
you might have shadowing here (and we should have a lint rule for this), but want to call this `innerSum`, or `_` and use the outer reduce sum?